### PR TITLE
replace moment with date-fns

### PR DESCRIPTION
**Pull request by killanch**
- added DateTimeUtils.js and making use of functions from same

### CODE REVIEW

**Comment by killanch** [comment]
@natabal @nxanil can you pls review?

File: Graphs/LineGraph/index.js [hunk 1]
```diff
@@ -12,20 +12,16 @@ import {
...
     timeFormat
 } from "d3";
-import moment from 'moment';
-import momentDuration from 'moment-duration-format';
 import isEqual from 'lodash/isEqual';
 import * as d3 from 'd3'
 import isEmpty from 'lodash/isEmpty';
 import ReactTooltip from 'react-tooltip';
-
+import {addMillisecs, formatDate} from "vis-graphs/utils/DateTimeUtils";
```
Comment (natabal) [inline Graphs/LineGraph/index.js:19]: please use relative path to the current location: .../utils/...

File: utils/columnAccessor.js [hunk 2]
```diff
@@ -50,7 +47,8 @@ const columnAccessor = ({ column, format, timeFormat, totalCharacters, duration
                 )
         }
     } else if (duration) {
-        return (d) => moment.duration(value(d)).format(duration);
+        //convert d to Date in milliseconds with timezone offset, and format the same using duration string passed
+        return (d) => formatDate(addMillisecs(new Date(new Date(0).getTimezoneOffset()*1000*60), value(d)), duration);
```
Comment (natabal) [inline utils/columnAccessor.js:51]: why all this when you can use date-fns duration?

File: Graphs/LineGraph/index.js [hunk 3]
```diff
@@ -454,7 +450,7 @@ class LineGraph extends XYGraph {
           .tickSizeOuter(yTickSizeOuter);
 
         if (yTickFormat || yTickFormat === "") {
-            const yAxisTickFormat = (yTickFormatType === duration) ? (d) => moment.duration(d).format(yTickFormat, { trim: false }) : format(yTickFormat);
+            const yAxisTickFormat = (yTickFormatType === duration) ? (d) => formatDate(addMillisecs(new Date(new Date(0).getTimezoneOffset()*1000*60), d), yTickFormat) : format(yTickFormat);
```
Comment (natabal) [inline Graphs/LineGraph/index.js:453]: why all this when you can use date-fns duration? Why did you decide on this?

Comment (killanch) [reply Graphs/LineGraph/index.js:453]: This code formats the provided milliseconds(d) duration as per the format string (yTickFormat ) provided.
There is no equivalent date-fns API for this.
I already explored the below APIs, but found them unsuitable for this,above purpose, hence the round-about approach.
https://date-fns.org/v2.15.0/docs/formatDistanceStrict
https://date-fns.org/v2.15.0/docs/formatDuration

**Comment by killanch** [comment]
@natabal addressed comments, this is now ready for review

**Comment by killanch** [comment]
@natabal pls merge, I do not have merge privileges on this repo

**Pull request merged**